odxtools/loadfile.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: MIT
+import os
+from pathlib import Path
+
+from .database import Database
+
+
+def load_pdx_file(pdx_file: str) -> Database:
+    db = Database()
+    db.add_pdx_file(pdx_file)
+    db.refresh()
+    return db
+
+
+def load_odx_d_file(odx_d_file_name: str) -> Database:
+    db = Database()
+    db.add_odx_file(odx_d_file_name)
+    db.refresh()
+
+    return db
+
+
+def load_file(file_name: str) -> Database:
```

Comment:
`load_file` feels like a useless alias of load_files

Reply:
yeah, but we should try not to break too many people's code. (and using `load_file()` for multiple files feels a bit stange to me...)